src/aleph/netty.clj

```diff
@@ -1298,10 +1331,25 @@ initialize an DnsAddressResolverGroup instance.
            java.io.Closeable
            (close [_]
              (when (compare-and-set! closed? false true)
+               ;; This is the three step closing sequence:
+               ;; 1. Stop listening to incoming requests
                (-> ch .close .sync)
-               (-> group (.shutdownGracefully (long shutdown-quiet-period)
-                                              (long shutdown-timeout)
-                                              TimeUnit/SECONDS))
+               (-> (if (pos? shutdown-timeout)
```

Comment:
All the other integration tests are damn slow on Aleph now.
I would consider adding `shutdown-timeout 0` everywhere else where we are starting a server.

Reply:
Ok will do (much later today though)

Reply:
done